### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Edited for clarity and grammar, adjusted formatting and headings for consistency.

### CODE REVIEW

**Review by jasonblais — DISMISSED** [review]

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ description: >-
...
+At Mattermost, we want you to be successful Day 1. As staff members, you are empowered to determine and purchase what you need in order to do your job. We value high-impact over stringent expense rules and these expense guidelines are intended to help make decisions not hinder movement. Depending on country/region, some items may be way below or way above the averages listed. If you need to spend more to get the right equipment for you to be productive, we trust you’re making purchase decisions to optimize cost and efficiency. If you are uncertain, feel free to reach out in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
At Mattermost, we want you to be successful from Day 1. As staff members, you are empowered to determine and purchase what you need in order to do your job. We value high impact over stringent expense rules and these expense guidelines are intended to help make decisions not hinder movement. Depending on country/region, some items may be way below or way above the averages listed. If you need to spend more to get the right equipment for you to be productive, we trust you’re making purchase decisions to optimize cost and efficiency. If you are uncertain, feel free to reach out in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel.
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 2]
```diff
@@ -6,16 +6,16 @@ description: >-
...
 
 ## Mattermost Guidelines
 
-Mattermost’s policy for spending company money is influenced by GitLab, who [open sources its policy on spending company money online](https://about.gitlab.com/handbook/spending-company-money/) and who has been [highly successful in a open, frugal, iterative culture](https://about.gitlab.com/2019/09/17/gitlab-series-e-funding/),
+Mattermost’s policy for spending company money is influenced by GitLab, who [open sources its policy on spending company money online](https://about.gitlab.com/handbook/spending-company-money/) and who has been [highly successful in an open, frugal, iterative culture](https://about.gitlab.com/2019/09/17/gitlab-series-e-funding/),
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
Mattermost’s policy for spending company money is influenced by GitLab, who [open sources its policy on spending company money online](https://about.gitlab.com/handbook/spending-company-money/) and who has been [highly successful in an open, frugal, iterative culture](https://about.gitlab.com/2019/09/17/gitlab-series-e-funding/).
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 3]
```diff
@@ -6,16 +6,16 @@ description: >-
...
 
 1. Spend company money like it is your own money. 
 2. You [don't have to ask permission](https://m.signalvnoise.com/you-dont-have-my-permission/) before making purchases on behalf of the company. When in doubt, inform your manager before the purchase or as soon as possible after the purchase. 
-3. It is uncommon for you to need all of the items listed below. Use your best judgement and buy them as you need them. If you wonder if something is common, feel free to ask [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team) \(and in turn, People Ops should update the list\).
-4. You may privately use Mattermost property, a laptop for example, to check your private e-mails or watch a movie as long as it does not violate the law, harm Mattermost, or interfere with Intellectual Property.
+3. It's uncommon for you to need all of the items listed below. Use your best judgement and buy them as you need them. If you wonder if something is common, feel free to ask [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team) \(and in turn, People Ops should update the list\).
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
3. It's uncommon for you to need all of the items listed below. Use your best judgement and buy them as you need them. If you wonder whether something is considered common, feel free to ask [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team) \(and in turn, People Ops should update the list\).
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 4]
```diff
@@ -6,16 +6,16 @@ description: >-
...
+4. You may privately use Mattermost property, a laptop for example, to check your private emails or watch a movie as long as it does not violate the law, harm Mattermost, or interfere with Intellectual Property.
 5. It is generally easiest and fastest for you to make any purchases for office supplies yourself and expense them. If you are unable to pay for any supplies yourself, please reach out to [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team).
 6. If you make a purchase that will cost Mattermost $1000 USD \(or over\), this is classified as company property, you will be required to return the item\(s\) if you leave the company.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
6. If you make a purchase that will cost Mattermost 1000 USD \(or over\), this is classified as company property, you will be required to return the item\(s\) if you leave the company.
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 5]
```diff
@@ -56,25 +56,25 @@ While we're sharing average costs for various items below, you should use your b
...
 
 **Non-Developers**
 
 * **Lenovo** - Most of our customers are Windows-based and so we recommend Windows-based machines, such as the [Lenovo X1 Carbon](https://www.lenovo.com/us/en/laptops/thinkpad/thinkpad-x/ThinkPad-X1-Carbon-6th-Gen/p/22TP2TXX16G) or similar for work machines. 
-* **Macbook 13”**- For non-developers choosing Macs we recommend getting a [MacBook Air 13-inch retina display with 256GB of storage and 8GB of memory](https://www.apple.com/shop/buy-mac/macbook-air/space-gray-1.6ghz-dual-core-processor-with-turbo-boost-up-to-3.6ghz-256gb#) or similar.
+* **MacBook 13”** - For non-developers choosing Macs we recommend getting a [MacBook Air 13-inch retina display with 256GB of storage and 8GB of memory](https://www.apple.com/shop/buy-mac/macbook-air/space-gray-1.6ghz-dual-core-processor-with-turbo-boost-up-to-3.6ghz-256gb#) or similar.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
* **MacBook 13”** - For non-developers choosing Macs we recommend getting a [MacBook Air 13-inch retina display with 256 GB of storage and 8 GB of memory](https://www.apple.com/shop/buy-mac/macbook-air/space-gray-1.6ghz-dual-core-processor-with-turbo-boost-up-to-3.6ghz-256gb#) or similar.
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 6]
```diff
@@ -56,25 +56,25 @@ While we're sharing average costs for various items below, you should use your b
...
 
 **Developers**
 
-* **Macbook 15”**- For developers choosing to use Macs, we recommend getting a [MacBook Pro 15-inch retina with 512GB of storage and 32GB of memory](https://www.apple.com/shop/buy-mac/macbook-pro/15-inch-space-gray-2.6ghz-6-core-512gb) or similar. As an example, other teams may not require 32GB memory or 512GB of storage. 
-* **PC** - Any suitable laptop with comparable specifications and lower or similar cost to the recommended MacBook. Many of our engineers use Lenovo laptops from the Thinkpad X, T or P series. If in doubt, we recommend a Lenovo Thinkpad T series.
+* **MacBook 15”** - For developers choosing to use Macs, we recommend getting a [MacBook Pro 15-inch retina with 512GB of storage and 32GB of memory](https://www.apple.com/shop/buy-mac/macbook-pro/15-inch-space-gray-2.6ghz-6-core-512gb) or similar. As an example, other teams may not require 32GB memory or 512GB of storage.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
* **MacBook 15”** - For developers choosing to use Macs, we recommend getting a [MacBook Pro 15-inch retina with 512 GB of storage and 32 GB of memory](https://www.apple.com/shop/buy-mac/macbook-pro/15-inch-space-gray-2.6ghz-6-core-512gb) or similar. As an example, other teams may not require 32 GB memory or 512 GB of storage. 
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 7]
```diff
@@ -167,23 +167,23 @@ The following links provide further detail regarding location-specific per diems
 * [GSA annual per diem rates for the US](https://www.gsa.gov/travel/plan-book/per-diem-rates/per-diem-rates-lookup)
 * [Foreign Per Diem Rates by Location administered by the US Department of State](https://aoprals.state.gov/web920/per_diem.asp)
 
-When dining with another staff member on one check, the senior most person is responsible for payment.  In submission of your receipt, include the names of all staff members who attended in Expensify documentation.
+When dining with another staff member on one check, the senior most person is responsible for payment. When submitting your receipt in Expensify, include the names of all staff members who attended.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
When dining with another staff member on one check, the most senior person is responsible for payment. When submitting your receipt in Expensify, include the names of all staff members who attended.
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 8]
```diff
@@ -167,23 +167,23 @@ The following links provide further detail regarding location-specific per diems
...
+Please spend company money as if it's your own. As always, security and safety is important and paramount for our staff members. 
 
 #### Loyalty Points 
 
 Staff members may retain points earned for hotel stays. However, hotel reservations should optimize business needs and cost vs. points. 
 
-### **MILEAGE REIMBURSEMENT**
+### MILEAGE REIMBURSEMENT
 
 We will reimburse mileage according to local laws of the country your contract is legally signed. For example, a contract with Mattermost, Inc. would be subject to IRS mileage reimbursement at the [US rate per mile](http://www.irs.gov/Tax-Professionals/Standard-Mileage-Rates). For approval, you must add a screenshot of a map to the expense in Expensify indicating the mileage.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
We will reimburse mileage according to local laws of the country in which your contract is legally signed. For example, a contract with Mattermost, Inc. would be subject to IRS mileage reimbursement at the [US rate per mile](http://www.irs.gov/Tax-Professionals/Standard-Mileage-Rates). For approval, you must add a screenshot of a map to the expense in Expensify indicating the mileage.
```

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 9]
```diff
@@ -218,16 +218,16 @@ Flexible office spaces or hot desk will be reviewed based on average costs for t
...
 
-1. Reach out to your manager/department head regarding the approval
-2. Give the business reason need and \(if applicable\) date/s if approval relates to an event or conference
-3. If the manager approves this as legitimate business need, the manager is required to contact Finance for spending approval at [expenses@mattermost.com](mailto:expenses@mattermost.com)
-4. If/When approved, approval email will be sent to the staff member and Manager
-5. Staff member must attach the approval to the Expensify expense report
+1. Reach out to your manager/department head regarding the approval.
+2. Give the business reason need and \(if applicable\) date/s if approval relates to an event or conference.
+3. If the manager approves this as legitimate business need, the manager is required to contact Finance for spending approval at [expenses@mattermost.com](mailto:expenses@mattermost.com).
+4. If/when approved, approval email will be sent to the staff member and manager.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
4. If/when approved, an approval email will be sent to the staff member and manager.
```

**Comment by jasonblais** [comment]
@justinegeffen I'm thinking if this is just for grammar and formatting, we could probably merge without a review from Isabel/Tim.

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 1 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 2 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 3 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 4 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 5 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 6 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: Approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 7 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 8 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: Approved

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 9 above)
Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: approved

**Comment by TQuock** [comment]
@justinegeffen I have reviewed and approved all updates

**Comment by justinegeffen** [comment]
Thanks, @TQuock! I think we're good to merge with one approval. :)

**Pull request merged**